x-pack/plugins/fleet/kibana.jsonc

```diff
@@ -35,6 +35,7 @@
       "telemetry",
       "discover",
       "ingestPipelines",
+      "slo",
```

Comment:
@shahzad31 I just started looking into that. At the moment I don't know how the fleet/integration works, but I don't see any machine learning module added as a dependency to fleet plugin. The reason I mention an ml module, is because I was looking into [Adding ML module asset type](https://github.com/elastic/kibana/pull/94950) into Fleet as an example and I didn't find any dependency between 2 plugins.